client/extensions/woocommerce/state/wc-api/reducer.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Internal dependencies
+ */
+import error from './error-reducer';
+import {
+	SERIALIZE,
+	DESERIALIZE,
+} from 'state/action-types';
+
+const initialState = {};
+
+const handlers = {
+	...error,
+};
+
+export default function( state = initialState, action ) {
+	const { type, payload } = action;
+	const { siteId } = payload || {};
+	const handler = handlers[ type ];
+
+	if ( handler ) {
+		// Necessary to ensure this data is not persisted.
+		if ( type === SERIALIZE || type === DESERIALIZE ) {
```

Comment:
Shouldn't we be checking for these types regardless of whether an error handler was registered in case our reducer is combined with others that do have actions of these types?